[2023-06-27T15:43:20.839+0000] {taskinstance.py:1090} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: get_pnadc_data_new.transfer_s3_to_rds manual__2023-06-27T11:42:42-04:00 [queued]>
[2023-06-27T15:43:20.848+0000] {taskinstance.py:1090} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: get_pnadc_data_new.transfer_s3_to_rds manual__2023-06-27T11:42:42-04:00 [queued]>
[2023-06-27T15:43:20.848+0000] {taskinstance.py:1288} INFO - 
--------------------------------------------------------------------------------
[2023-06-27T15:43:20.848+0000] {taskinstance.py:1289} INFO - Starting attempt 1 of 1
[2023-06-27T15:43:20.849+0000] {taskinstance.py:1290} INFO - 
--------------------------------------------------------------------------------
[2023-06-27T15:43:20.890+0000] {taskinstance.py:1309} INFO - Executing <Task(S3ToSqlOperator): transfer_s3_to_rds> on 2023-06-27 15:42:42+00:00
[2023-06-27T15:43:20.897+0000] {standard_task_runner.py:55} INFO - Started process 5295 to run task
[2023-06-27T15:43:20.900+0000] {standard_task_runner.py:82} INFO - Running: ['***', 'tasks', 'run', 'get_pnadc_data_new', 'transfer_s3_to_rds', 'manual__2023-06-27T11:42:42-04:00', '--job-id', '66', '--raw', '--subdir', 'DAGS_FOLDER/get_pnadc_data_new.py', '--cfg-path', '/tmp/tmph0d6g3gl']
[2023-06-27T15:43:20.903+0000] {standard_task_runner.py:83} INFO - Job 66: Subtask transfer_s3_to_rds
[2023-06-27T15:43:20.990+0000] {task_command.py:389} INFO - Running <TaskInstance: get_pnadc_data_new.transfer_s3_to_rds manual__2023-06-27T11:42:42-04:00 [running]> on host fef15d81d3ea
[2023-06-27T15:43:21.187+0000] {taskinstance.py:1518} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=get_pnadc_data_new
AIRFLOW_CTX_TASK_ID=transfer_s3_to_rds
AIRFLOW_CTX_EXECUTION_DATE=2023-06-27T15:42:42+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2023-06-27T11:42:42-04:00
[2023-06-27T15:43:21.189+0000] {s3_to_sql.py:101} INFO - Loading testfiles/processed/processed_data_pnadc_2023_06_27.csv to SQL table people_info...
[2023-06-27T15:43:21.206+0000] {base.py:73} INFO - Using connection ID 'aws_conn' for task execution.
[2023-06-27T15:43:21.206+0000] {connection_wrapper.py:337} INFO - AWS Connection (conn_id='aws_conn', conn_type='aws') credentials retrieved from login and password.
[2023-06-27T15:43:25.236+0000] {base.py:73} INFO - Using connection ID 'postgres_conn' for task execution.
[2023-06-27T15:43:25.457+0000] {base.py:73} INFO - Using connection ID 'postgres_conn' for task execution.
[2023-06-27T15:43:47.857+0000] {sql.py:473} INFO - Done loading. Loaded a total of 100 rows into people_info
[2023-06-27T15:43:47.913+0000] {taskinstance.py:1332} INFO - Marking task as SUCCESS. dag_id=get_pnadc_data_new, task_id=transfer_s3_to_rds, execution_date=20230627T154242, start_date=20230627T154320, end_date=20230627T154347
[2023-06-27T15:43:48.004+0000] {local_task_job.py:212} INFO - Task exited with return code 0
[2023-06-27T15:43:48.033+0000] {taskinstance.py:2596} INFO - 1 downstream tasks scheduled from follow-on schedule check
